### PR TITLE
Make beat directory configurate in Makefile

### PR DESCRIPTION
**Pull request by mrkschan**

### CODE REVIEW

File: libbeat/scripts/Makefile [hunk 1]
```diff
@@ -8,6 +8,7 @@ export GO15VENDOREXPERIMENT=1
 GOFILES = $(shell find . -type f -name '*.go')
 SHELL=/bin/bash
 ES_HOST?="elasticsearch-210"
+BEAT_DIR?=github.com/elastic/beat
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: It is `github.com/elastic/beats` instead of `beat` here, that is why the test are broken. Other LGTM.

**Comment by mrkschan** [comment]
Sorry for the careless mistake. That should have been fixed :)

**Comment by ruflin** [comment]
@mrkschan Thanks for the fix. Could you squash the commits into 1 commit?

**Comment by mrkschan** [comment]
Does it look good now? I didn't merge master yet, btw.

**Comment by ruflin** [comment]
LGTM. No need to merge master as there doesn't seem to be a conflict. Will merge it now, ok?

**Comment by mrkschan** [comment]
please :)

**Pull request merged**